tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -3204,7 +3300,7 @@ public function groupHasBeenCreatedOnDatabaseBackend($group) {
 	 * @throws \Exception
 	 */
 	public function theseGroupsHaveBeenCreated(TableNode $table) {
-		$this->verifyTableNodeColumns($table, ['groupname']);
+		$this->verifyTableNodeColumns($table, ['groupname'], ["comment"]);
```

Comment:
lets do the inverted commas the same:
```suggestion
		$this->verifyTableNodeColumns($table, ['groupname'], ['comment']);
```